dtreeviz/trees.py

```diff
@@ -1526,8 +1526,9 @@ def _ctreeviz_univar(shadow_tree,
         ax.set_title(title, fontsize=fontsize, color=colors['title'])
 
     if 'splits' in show:
+        split_heights = [*ax.get_ylim()]
```

Comment:
I didn't realize these could change height... wow. i guess the plot could do that.